.github/workflows/pull-request-labeler.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2020 Dremio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Auto Labeler
+
+on:
+  pull_request_target:
+
+jobs:
+  triage:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/labeler@v4
+        with:
+          repo-token: "${{ secrets.CI_REPORTS_TOKEN }}"
```

Comment:
should we set `sync-labels: true` ?

```
Whether or not to remove labels when matching files are reverted or no longer changed by the PR
```

iceberg is using it:
https://github.com/apache/iceberg/blob/master/.github/workflows/labeler.yml

Reply:
Was thinking about it, but what it you open a PR, add the label yourself, and just want to make sure that things you changed elsewhere still works.